### PR TITLE
Fix variable string substitutions

### PR DESCRIPTION
**Pull request by codenirvana**
Reference: https://github.com/postmanlabs/postman-app-support/issues/5682

### CODE REVIEW

File: test/unit/property.test.js [hunk 1]
```diff
@@ -215,9 +215,67 @@ describe('Property', function () {
...
+            return obj;
+        }
+
         it('should bail out if a non-string argument is passed', function () {
             expect(Property.replaceSubstitutions(['random'])).to.eql(['random']);
         });
+
+        it('should resolve poly chained variable 19 times', function () {
+            // eslint-disable-next-line max-len
+            var str = '{{20{{19{{18{{17{{16{{15{{14{{13{{12{{11{{10{{9{{8{{7{{6{{5{{4{{3{{2{{1{{0}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}',
```
Comment (kunagpal) [inline test/unit/property.test.js]: This will have to be updated if the limit ever changes.

File: lib/superstring/index.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ _.assign(SuperString.prototype, /** @lends SuperString.prototype */ {
         }));
 
         this.replacements = replacements; // store the last replacements
-        this.substitutions += Number(replacements); // if any replacement is done, count that some substitution was made
+        replacements && (this.substitutions += 1); // if any replacement is done, count that some substitution was made
```
Comment (kunagpal) [inline lib/superstring/index.js]: Is the substitutions property being used elsewhere?

File: test/unit/property.test.js (hunk 1 above)
Comment (codenirvana) [reply test/unit/property.test.js]: Yes, any alternative?

File: lib/superstring/index.js (hunk 2 above)
Comment (codenirvana) [reply lib/superstring/index.js]: No.

File: test/unit/property.test.js (hunk 1 above)
Comment (kunagpal) [reply test/unit/property.test.js]: Have a function that accepts a number argument and builds this string for you.

>

Comment (codenirvana) [reply test/unit/property.test.js]: done

**Pull request merged**